### ISSUE TITLE
data augmentation details

### ISSUE THREAD

**Issue by twangnh** [ljanyst/ssd-tensorflow#20]
Hi! ljanyst, thanks for sharing your wonderful work, for the result you posted, have you add the image expansion data augmentation?

**Comment by twangnh** [comment]
I find that expansion augmentation is implemented, so the current result is approximately 3 points below the original caffe implementation?

**Comment by ljanyst** [comment]
Yes, it's roughly 3-4 points below the Caffe implementation. As far as I can tell, the only two differences remaining between this code and the original implementation are the initial weights of the VGG16 base and the learning rate for biases. I will try to run some tests fixing the second difference during the weekend, although it only can be done in quite a pathological way in TensorFlow - requires two forward passes.

**Comment by twangnh** [comment]
For doubling learning rate of bias, why not just double the calculated gradients respectively? another thing is that the faster rcnn paper mentioned the border anchors can introduce hard to correct terms for training RPN, and ignores the border anchors when doing backprop, would this be a potential improvement for SSD training ?

**Comment by ljanyst** [comment]
Good point. I will see if I can do it that way.

**Comment by ljanyst** [comment]
Hmm, the SSD paper says that it leaves the border anchors untouched. It's also not cutting the anchors in the subsampled images, just has a binary filter for them based on the center location. I will have a look at what the implementation actually does. I would not be surprised if it's something different.

**Comment by twangnh** [comment]
hope you have a good result!

**Comment by twangnh** [comment]
Hi! @ljanyst , any update on your experiments?
